web/packages/teleport/src/Roles/RoleEditor/StandardEditor/standardmodel.ts

```diff
@@ -958,6 +969,24 @@ function kubernetesResourceToModel(
   };
 }
 
+export function mcpToolsToModel(
+  mcpPermissions: MCPPermissions,
+  pathPrefix: string
+): {
+  model: string[];
+  conversionErrors: ConversionError[];
+} {
+  const { tools, ...unsupported } = mcpPermissions;
+  const conversionErrors: ConversionError[] = [];
+  conversionErrors.push(
+    ...unsupportedFieldErrorsFromObject(pathPrefix, unsupported)
+  );
+  return {
+    model: tools || [],
+    conversionErrors,
```

Comment:
Nit: since there's only one place where you add to `conversionErrors`, you could skip initializing the `conversionErrors` variable and pushing to it and instead initialize this field directly with the result of `unsupportedFieldErrorsFromObject`.